src/app/(algorithms)/sorting-algorithms/bubble-sort/page.tsx

```diff
@@ -1,3 +1,39 @@
+"use client";
+import { useEffect, useState } from "react";
+import styles from "./bubble.module.css";
+
+const generateUniqueNumbers = (n: number): number[] => {
+  const uniqueNumbers = new Set<number>();
+  while (uniqueNumbers.size < n) {
+    const randomNumber = Math.floor(Math.random() * 91) + 10;
+    uniqueNumbers.add(randomNumber);
+  }
+  return Array.from(uniqueNumbers);
+};
+
 export default function Page() {
-  return <div>Bubble</div>;
+  const [numbers, setNumbers] = useState<number[]>([]);
+
+  useEffect(() => {
```

Comment:
it does not make any sense to set up it like this, just call the function in initial value of useState